dungeon/src/contrib/level/generator/graphBased/RoomBasedLevelGenerator.java

```diff
@@ -122,7 +122,7 @@ private static LevelSize sizeFor(final LevelNode node) {
    *
    * @param node Node to configure the doors for.
    */
-  private static void configureDoors(final LevelNode node) {
+  public static void configureDoors(final LevelNode node) {
```

Comment:
Diese Umstellung von `private` auf `public` **muss** als Änderung im ersten Kommentar des Pull-Requests erklärt werden: (a) Diese Änderung wird durch diesen PR gemacht, und (b) sie ist erforderlich, weil ...

Reply:
erster PR Kommentar ist aktualisiert

Reply:
Ja, aber das "Mit der neuen Struktur muss von außen auf diese Methode zugegriffen werden können." ist noch keine Erklärung. 

ich würde da irgendwas erwarten wie "In Klasse X wird diese Funktionalität benötigt zum XYZ."